nyx_client/nyx_client/client.py

```diff
@@ -583,6 +613,8 @@ def create_data(
             preview: A preview or sample of the data.
             price: The price of the data in cents. If 0, the data is free.
             license_url: The URL of the license for the data.
+            access_control: Either allow all, or allow none
```

Comment:
If that's the case, shouldn't the type hint use e.g. `Literal["all", "none"]? The end user shouldn't need to know about the exact IRIs required. (I appreciate you've added come constants but still.)

Reply:
Yeah I think for simplicity sake I will just go with the first suggestion. In my head this was to allow people to explicitly provide a did, but that's pretty advanced usuage, and you're right it's a bit too much overhead.

Reply:
:+1:  Nice one!

(unimportant) we have one more "fixed" enum style field - `status`. Since it has a default - if we don't want people to use it for now we could either remove it (given API breaking change anyway).. or explicitly state `draft` & `published` (esp since the server will reject anything else).
But then we'd probably need to document what the two mean and they overlap with `circles` & `access_control`.

@StephanieBracaloni  - what do you think we should do?

Reply:
I would leave it as it is for the scope of this PR. 
The removal (or the completion) of the status should be handled holistically.